src/stars.css

```
@@ -0,0 +1,34 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+
+}
+
+.stars--1 .stars__star:nth-child(1) {
```

```suggestion
.stars--1 .stars__star:nth-child(1),
.stars--2 .stars__star:nth-child(-n+2) ,
... {
```
Consider using commas to group selectors and avoid repeating code